docs/core/compatibility/sdk/6.0/implicit-namespaces-rc1.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Preview-to-preview breaking change: Implicit namespaces disabled by default"
+description: Learn about the preview-to-preview breaking change in .NET 6 where the implicit namespaces feature is disabled by default in C# projects.
+ms.date: 09/02/2021
+---
+# Implicit namespaces disabled by default
+
+C# 10.0 introduced [global namespace](../../../../csharp/language-reference/keywords/using-directive.md#global-modifier) support for C# projects. In Preview 7, [the .NET SDK implicitly added global namespaces](implicit-namespaces.md) to .NET projects, and the feature was enabled by default. In .NET 6 RC 1, implicit namespaces are disabled by default, and some of the associated MSBuild property and item names have changed. You may need to re-enable the feature or explicitly include namespaces that your project depends on.
+
+## Version introduced
+
+.NET 6 RC 1
+
+## Old behavior
+
+In .NET 6 Preview 7, the .NET SDK implicitly includes a set of default namespaces for C# projects that target .NET 6 or later and that use one of the following SDKs:
+
+- Microsoft.NET.Sdk
+- Microsoft.NET.Sdk.Web
+- Microsoft.NET.Sdk.Worker
+
+In addition, you could:
+
+- Disable the feature globally using the `DisableImplicitNamespaceImports` MSBuild property, or on an SDK-specific basis using the `DisableImplicitNamespaceImports_DotNet`, `DisableImplicitNamespaceImports_Web`, and `DisableImplicitNamespaceImports_Worker` properties.
+- Import specific namespaces using the `Import` item type.
+
+## New behavior
+
+Starting in .NET 6 RC 1, no implicit namespaces are included, by default. However, you can enable the feature in your C# project by setting the [`ImplicitUsings` MSBuild property](../../../project-sdk/msbuild-props.md#implicitusings) to `true` or `enable`.
+
+In addition, for C# projects:
```

Comment:
Might be worth mentioning here that new projects targeting .NET 6 have implicit usings enabled by default.